group-coordinator/src/main/java/org/apache/kafka/coordinator/group/CoordinatorRecordHelpers.java

```diff
@@ -106,6 +109,28 @@ public static CoordinatorRecord newMemberSubscriptionRecord(
         );
     }
 
+    /**
+     * Creates a ConsumerGroupMemberMetadata tombstone.
+     *
+     * @param groupId  The consumer group id.
+     * @param memberId The consumer group member id.
+     * @return The record.
+     */
+    public static CoordinatorRecord newMemberSubscriptionTombstoneRecord(
```

Comment:
Why was this not here before? It seems like this is something consumer specific.

Reply:
Nevermind! It was there before just a bit below.